examples/apps/quick_tap.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Quick Tap - tap your cube as fast as possible when the colors match, but never tap on red!
+
+The game ends when a player scores 5 points.
+'''
+import asyncio, random, sys, time
+
+import cozmo
+
+from cozmo.lights import blue_light, Color, green_light, Light, red_light, white_light, off_light
+from cozmo.util import degrees, distance_mm, radians, speed_mmps
+
+purple_light = Light(Color(name = 'purple', rgb = (255, 0, 255)))
+yellow_light = Light(Color(name = 'yellow', rgb = (255, 255, 0)))
+
+LIGHT_COLORS_LIST = [blue_light, green_light, purple_light, red_light, white_light, yellow_light]
+
+CHOOSE_CUBES_STATE = 'choose_cubes' # If the game is in CHOOSE_CUBES_STATE, on_cube_tap assigns the player's cube.
+GAME_STATE = 'game' # If the game is in GAME_STATE, on_cube_tap registers the tap time of the players.
+
+MAKE_BUZZERS_DIFFERENT_COLORS = 'MAKE_BUZZERS_DIFFERENT_COLORS'
+MAKE_BUZZERS_RED = 'MAKE_BUZZERS_RED'
+MAKE_BUZZERS_SAME_COLORS = 'MAKE_BUZZERS_SAME_COLORS'
+
+# The buzzers have a 50% chance of displaying the same colors.
+RATE_MAKE_BUZZERS_DIFFERENT_COLORS = 0.17 # The buzzers have a 17% chance of displaying different colors.
+RATE_MAKE_BUZZERS_RED = 0.33 # the buzzers have a 33% chance of displaying red.
+
+RATE_COZMO_ACCURACY = 0.9 # Cozmo has a 90% chance of reacting correctly to the buzzers.
+# This number can therefore be lowered to have Cozmo more frequently make the wrong move.
+
+SCORE_TO_WIN = 5 # the game ends once either player's score has reached SCORE_TO_WIN
+
+class QuickTapGame:
+    '''The game logic of Quick Tap.'''
+    def __init__(self, robot: cozmo.robot.Robot):
+        self.robot = robot
+        self.player = QuickTapPlayer()
+        self.cozmo_player = CozmoQuickTapPlayer(robot)
+        robot.add_event_handler(cozmo.anim.EvtAnimationCompleted, self.on_anim_completed)
+        robot.add_event_handler(cozmo.objects.EvtObjectTapped, self.on_cube_tap)
+
+        self.cubes = None
+        self.countdown_cube = None
+
+        self.buzzer_display_type = None
+
+        self.round_start_time = time.time()
+        self.quick_tap_player_1 = None
+        self.quick_tap_player_2 = None
+        self.round_over = False
+
+        self.quick_tap_state = CHOOSE_CUBES_STATE
+
+    async def cozmo_ready_position(self):
```

Comment:
suggest renaming to make it clear that this does something, and doesn't e.g. just return a "ready position" - how about "move_cozmo_to_ready_pose"